OurUmbraco/Community/GitHub/Controllers/GitHubContributorController.cs

```diff
@@ -121,7 +120,7 @@ public List<GitHubGlobalContributorModel> GetContributors()
 
             string[] login = System.IO.File.ReadAllLines(configPath).Where(x => x.Trim() != "").Distinct().ToArray();
 
-            var githubController = new GitHubController();
+            var githubController = new GitHubService();
```

Comment:
I haven't updated the controller yet (besides what Resharper did when renaming the class). But yes, the variable name should reflect the new name of the class.